web-app/src/components/tournament/creator/selectableTournamentType.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="row col-8 col-sm-4 justify-center q-ma-md">
+    <q-btn class="row col-12" no-caps padding="none">
+      <q-card
+        flat
+        class="col-12 tournamentTypeCard"
+        @click="updateTournamentType"
+      >
+        <q-icon class="col-12" :name="iconName" style="font-size: 1000%" />
+        <div class="text-h6">{{ tournamentTypeName }}</div>
+      </q-card>
+    </q-btn>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import EventBus from '../../../services/EventBus';
+
+@Component
+export default class SelectableTournamentType extends Vue {
+  @Prop({ type: String, required: true }) readonly tournamentTypeName!: string;
+  @Prop({ type: String, required: true }) readonly tournamentType!: string;
+  @Prop({ type: String, required: true }) readonly iconName!: string;
+
+  private updateTournamentType() {
+    EventBus.$emit('updateActiveTournamentType', this.tournamentType);
```

Comment:
I wonder if this is a good use of the EventBus. You can emit this event to parent component and that parent can forward emit to parent of the parent 
As name of this event I suggest to use `select` like in selectors

You will be able to receive this as e.q. 
```javascript
<component
   @myCustomEvent="forwardEvent"
/>

...

<script>
...
forwardEvent(data) {this.$emit('myCustomEvent', data}
```

It's not every important but would be nice to code this as above, maybe in correction task that You mentioned in the description

Reply:
done so after consulting with @eastemployeer 
but probably will do in the mentioned task